frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/CodeSample.jsx

```diff
@@ -37,7 +37,7 @@ export default class CodeSample extends Component {
     const selected = _.findWhere(options, { name });
     const source = selected && selected.source();
     return (
-      <div className={className}>
+      <div className={className} data-testid={`${dataTestId}-code-sample`}>
```

Comment:
The `data-testid` that's passed into `CodeSample` is only used as a prefix for the other test IDs within this component, so putting this ID at the top level matches the pattern here already. It might make more sense to keep this as just `data-testid={dataTestId}`, I'm open to suggestions